controllers/openstackdataplanedeployment_controller.go

```diff
@@ -226,13 +234,14 @@ func (r *OpenStackDataPlaneDeploymentReconciler) Reconcile(ctx context.Context,
 		}
 
 		deployer := deployment.Deployer{
-			Ctx:             ctx,
-			Helper:          helper,
-			NodeSet:         &nodeSet,
-			Deployment:      instance,
-			Status:          &instance.Status,
-			AeeSpec:         &ansibleEESpec,
-			InventorySecret: nodeSetSecretInv,
+			Ctx:                         ctx,
+			Helper:                      helper,
+			NodeSet:                     &nodeSet,
+			Deployment:                  instance,
+			Status:                      &instance.Status,
+			AeeSpec:                     &ansibleEESpec,
+			InventorySecrets:            globalInventorySecrets,
+			AnsibleSSHPrivateKeySecrets: globalSshKeySecrets,
```

Comment:
can you use consistent camelCase? it should be SSH everywhere.

Reply:
yes.